BridgeEmulator/HueEmulator3.py

```diff
@@ -457,7 +457,6 @@ def daylightSensor():
     if bridge_config["sensors"]["1"]["modelid"] != "PHDL00" or not bridge_config["sensors"]["1"]["config"]["configured"]:
         return
     # this doesn't even work???
```

Comment:
does this comment refer to the failing import? If so, I could delete it too...

Reply:
from my tests this is working fine. The conditions where made by me. First i add the check to see if the configuration is recent because daylight logical sensor was added later in the hue api. Older configuration use sensor id "1" for other things. Second condition check if the geo location where configured. Without them the astral library don't know for which location must calculate the sunrise and the sunset.

Reply:
Yep, I believe that comment was in relation to that import. Thanks!